### PR TITLE
Fix msgpack dependencies

### PR DESCRIPTION
**Pull request by jpdeplaix**
msgpack doesn't really depend on camlp4. It only needs it to execute tests and yet camlp4 doesn't need to be checked at configure time because the test uses a camlp4 library.

cc @mzp

### CODE REVIEW

**Comment by mzp** [comment]
thanks!

File: packages/msgpack/msgpack.1.2.1/opam [hunk 1]
```diff
@@ -14,7 +17,10 @@ install: [
 remove: [
   ["ocamlfind" "remove" "msgpack"]
 ]
-depends: ["ocamlfind" "camlp4"]
+depends: [
+  "ocamlfind"
```
Comment (yallop) [inline packages/msgpack/msgpack.1.2.1/opam]: Could `ocamlfind` also be marked `{build}` here?

Comment (jpdeplaix) [inline packages/msgpack/msgpack.1.2.1/opam]: Nope, it is also used in the `remove` section.

Comment (yallop) [inline packages/msgpack/msgpack.1.2.1/opam]: I don't think that's a problem: `{build}` just means "don't recompile the package when the dependency is updated".

**Pull request merged**